scripts/stats.js

```diff
@@ -92,18 +92,13 @@ function handleRoomStat(stats, res) {
 
 // Handles command stat 
 function handleCommandStat(stats, res) {
-    // If the command is not actually a command or the room is a private channel, exit
+    // If the room is not a public channel, exit
     var command = res.message.text;
     var room = res.message.room;
-    if ((command[0] != '!' && room[0] != 'D') || room[0] == 'G') {
```

Comment:
don't we need to keep the check that the command starts with an exclamation mark?